tests/controllers/ControllerFactory.ts

```diff
@@ -0,0 +1,29 @@
+import { Connection } from 'typeorm';
+import FeedbackService from '../../services/FeedbackService';
+import { FeedbackController } from '../../api/controllers/FeedbackController';
+import { UserController } from '../../api/controllers/UserController';
+import UserAccountService from '../../services/UserAccountService';
+import StorageService from '../../services/StorageService';
+
+export class ControllerFactory {
+  private static userController: UserController = null;
+
+  private static feedbackController: FeedbackController = null;
+
+  public static user(conn: Connection): UserController {
+    if (!ControllerFactory.userController) {
+      const userService = new UserAccountService(conn.manager);
+      const storageService = new StorageService();
```

Comment:
As an aside, we should prob pass in the S3 config to the `StorageService` constructor instead of importing.

Reply:
I personally think it's fine as is, just so that different services/controllers don't need to know about lower-level aws configs when instantiating the storage service, and so that aws doesn't need to get imported everywhere